api/src/search/service.ts

```diff
@@ -25,11 +25,16 @@ export class SearchService {
     });
   }
 
-  public search = async (query: string): Promise<SearchResults> => {
-    this.logger.info({ message: `Searching for ${query}` });
-    return {
-      results: [],
-    };
+  public search = async (q: string, limit?: number): Promise<SearchResults> => {
+    this.logger.info({ message: `Searching for "${q}" in all indexes` });
+    const searchResults = await this.meilisearch.multiSearch({
+      queries: [
+        { indexUid: "project", q, limit },
```

Comment:
let's expose only what we need using [attributes-to-retrieve](https://www.meilisearch.com/docs/reference/api/search#attributes-to-retrieve):
```suggestion
        { indexUid: "project", q, limit, attributesToRetrieve: ["id", "name"] }, },
```